tools/db_bench_tool.cc

```diff
@@ -5634,6 +5643,7 @@ class Benchmark {
     std::unique_ptr<const char[]> key_guard;
     Slice key = AllocateKey(&key_guard);
     PinnableSlice pinnable_val;
+    std::vector<PinnableSlice> pinnable_vals;
```

Comment:
Should this be initialized to include one (or more) values?  The first time through the operation will fail as we are looking for 0 entries.  I wonder if a reasonable default might make more sense.

Reply:
Sure, done (8).